npm/nwpolicy.go

```diff
@@ -189,3 +198,35 @@ func (npMgr *NetworkPolicyManager) DeleteNetworkPolicy(npObj *networkingv1.Netwo
 
 	return nil
 }
+
+func createCidrsRule(ingressOrEgress string, policyName string, ipsetEntries [][]string, ipsMgr *ipsm.IpsetManager) {
```

Comment:
what about moving `createCidrsRule` and `removeCidrsRule` to `ipsm.IpsetManager`?